backend/stylists/app/Http/Controllers/NotificationsController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use Illuminate\Support\Facades\Auth;
+use Validator;
+use App\Recommendation;
+use App\Push;
+use App\Client;
+use App\Stylist;
+use App\Look;
+use App\Product;
+
+class NotificationsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request, $action, $id = null, $action_id = null)
+    {
+        $method = strtolower($request->method()) . strtoupper(substr($action, 0, 1)) . substr($action, 1);
+        if ($id) {
+            $this->resource_id = $id;
+        }
+        if ($action_id) {
+            $this->action_resource_id = $action_id;
+        }
+        return $this->$method($request);
+    }
+
+    public function postPushNotifications(Request $request)
+    {
+        $app_section = isset($request->app_section) && $request->app_section != "" ? $request->app_section : "3";
+        $entity_type_id = isset($request->entity_type_id) ? $request->entity_type_id : '';
```

Comment:
If $entity_type_id is '' then no activity should be done. Instead an error message should be generated.